src/main/resources/avro/bdg.avdl

```diff
@@ -286,6 +299,13 @@ record Fragment {
    */
   array<Sequence> sequences = [];
   array<AlignmentRecord> alignments = [];
+
+
+  /**
+   Platform-specific information about this Fragment.
+   */
+  union { null, Platform } platformType;
+  union { null, IlluminaHTInfo } platformInfo;
```

Comment:
You don't.

The `Fragment` is one piece of DNA attached to one position in the flowcell.  The sequencing technology may be able to read the entire thing, generating a single sequence (hypothetically, something like nanopore tech), or it may read multiple parts generating multiple sequences (e.g., Illumina tech generating multiple reads, which are different parts of the same DNA fragment).

With this definition, reads that come from different parts of the flowcell would be sequences read from different `Fragments`.